### PR TITLE
Refactor directions

### PR DESCRIPTION
**Pull request by darya42185**
develop

https://github.com/ita-social-projects/dokazovi-fe/issues/132
[#132 Story]( )

### CODE REVIEW

File: src/lib/constants/direction-properties.ts [hunk 1]
```diff
@@ -1,58 +1,10 @@
-/* eslint-disable import/no-cycle */
-import { IDirection } from '../types';
+import { useSelector } from 'react-redux';
+import { RootStateType } from '../../store/rootReducer';
 
-export type DirectionPropertiesType = {
-  [key: string]: IDirection;
-};
+export const DirectionProperties = () => {
```
Comment (calster00) [inline src/lib/constants/direction-properties.ts:4]: we don't need this file anymore, don't we?

Comment (darya42185) [reply src/lib/constants/direction-properties.ts:4]: yes, I just forgot to delete. Thank you!

File: src/App.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
```
Comment (SergeOlabin) [inline src/App.tsx:1]: remove

File: src/lib/utilities/API/types.ts [hunk 3]
```diff
@@ -83,5 +83,14 @@ export type PostPostResponseType = {
   name: string;
 }
 
+export type PostDirectionType = {
```
Comment (SergeOlabin) [inline src/lib/utilities/API/types.ts:86]: it's better
```suggestion
export type DirectionType = {
```

`Post` refers to POST request when comparing with `GetDirectionType ` next ot it..

**Review by SergeOlabin — APPROVED** [review]
pls fix minor comments, the rest is good! 👍

**Review by calster00 — APPROVED** [review]

**Pull request merged**